### PR TITLE
This branch is to add Markup and styles along with JavaScript for the mobile menu

### PR DESCRIPTION
**Pull request by thenarain**
update Markup in nav-bar section
Updted styles for the mobile menu
Add script to display mobile-menu
Refactor entire nav-section

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -78,16 +40,60 @@
...
   width: 100%;
-  height: 100%;
+  height: 100vh;
   position: absolute;
-  background-color: #2230d2;
-  opacity: 0.8;
+  background-color: #6070FF;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(8px);
+}
```
Comment (Lordkaito) [inline style.css]: - [ ] When you show your menu on the mobile version, please make sure that the text is shown properly and you can distinguish the words. If you check the image below, you will see how your `I'm Mariana` text is being shown on your menu and not allowing the user to properly see the options, so please take a look at this issue.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @thenarain, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (thenarain) [reply style.css]: Thank you for the review. I have done the changes.

**Review by prolajumokeoni — APPROVED** [review]
Hi @thenarain, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved-stamp


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**